classes/local/v1p2/conformance.php

```diff
@@ -0,0 +1,586 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * One Roster Enrolment Client.
+ *
+ * @package    enrol_oneroster
+ * @copyright  Andrew Nicols <andrew@nicols.co.uk>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace enrol_oneroster\local\v1p2;
+
+use enrol_oneroster\local\conformance as conformance_base;
+
+/**
+ * One Roster v1p1 conformance tests.
```

Comment:
This comment is wrong, this is v1p2.

Also, is there a reason this cant just be a case of v1p2 extends v1p1, instead of both extending off of the base? That would remove all this code duplication